docs/databases/redis/redis-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,189 @@
+contributor:
+  name: Sergey Pariev
+  link: https://twitter.com/spariev
+external_resources:
+ - '[Redis Project Home Page](http://redis.io/)'
+ - '[Redis Configuration](http://redis.io/topics/config)'
+ - '[Redis Persistence](http://redis.io/topics/persistence)'
+ - '[Redis Security](http://redis.io/security)'
+---
+
+Redis is an open source in-memory data structure store, with optional disk writes for persistence, which can be used as key-value database, cache and message broker. Redis features built-it transactions, replication, and support for a variety of data structures such as strings, hashes, lists, sets and others. Redis can be made highly available with Redis Sentinel and supports automatic partitioning with Redis Cluster. This document provides both instructions for deploying the Redis server and an overview of best practices for maintaining Redis instances.
+
+## Before You Begin
+
+1.  Familiarize yourself with our [Getting Started](/docs/getting-started) guide and complete the steps for setting your Linode's hostname and timezone.
+
+2.  This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server) to create a standard user account, harden SSH access and remove unnecessary network services.
+
+3.  Update your system.
+
+		sudo apt-get update && sudo apt-get upgrade
+
+
+{: .note}
+>
+>This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you’re not familiar with the `sudo` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+
+## Redis Installation
+
+Redis package in Ubuntu 14.04 repository is outdated and misses several security patches, so in this case it is better to use third-party PPA.
+
+1.  Add Redis PPA repository to install the latest version -
+
+		sudo add-apt-repository ppa:chris-lea/redis-server
+
+2.  Update packages and install `redis-server` package
+
+		sudo apt-get update
+		sudo apt-get install redis-server
+
+3.  Verify Redis is up by running `redis-cli` and executing `ping` command
+
+		redis-cli
+		127.0.0.1:6379> ping
+		PONG
+
+Press **Ctrl-C** to exit from `redis-cli` prompt.
+
+
+## Redis Configuration
+
+### Configure Persistence Options
+
+Redis provides two options for disk persistence:
+
+* Point-in-time snapshots of the dataset, made at specified intervals (RDB)
+* Append only logs of all the write operations performed by the server (AOF).
+
+Each option has it's own pros and cons, which are described in great detail in Redis documentation. For most possible data safety it is advised to use both persistence methods.
+
+As the point-in-time snapshot persistence is enabled by default, the only thing you will need to do is to setup AOF persistence.
+For this, add the following lines to `redis.conf` file:
+
+{: .file-excerpt }
+/etc/redis/redis.conf
+:   ~~~
+    appendonly yes
```

Comment:
I'll reword this section, in lines of 
> make sure `appendonly` setting is set to `yes` and `appendfsync` is set to  `everysec`

as these settings are indeed present in default configuration file, but with other values.